repose-aggregator/components/filters/api-validator-filter/src/main/java/org/openrepose/filters/apivalidator/ApiValidatorHandler.java

```diff
@@ -153,9 +146,16 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
                 }
 
                 if (!isValid) {
-                    if (mbcsInvalidRequests != null) {
+                    if (Optional.ofNullable(metricsService).isPresent()) {
+                        MetricRegistry metricRegistry = metricsService.getRegistry();
                         for (String s : matchedRoles) {
-                            mbcsInvalidRequests.mark(s);
+                            metricRegistry.meter(
+                                    metricsService.name(
+                                            "org.openrepose.core.filters.ApiValidator",
+                                            "api-validator", // TODO replace "api-validator" with filter-id or name-number in sys-model
```

Comment:
Yep. I completely agree. There was just already enough going on with updating the metrics library, I was just trying to keep it all functionally equivalent.